src/components/vr_module/src/vr_module.cc

```diff
@@ -111,10 +116,34 @@ void VRModule::EmitEvent(const vr_mobile_api::ServiceMessage& message) {
       vr_mobile_api::RPCName>::instance()->raise_event(event);
 }
 
-bool VRModule::SendToMobile(const vr_mobile_api::ServiceMessage& message) {
+bool VRModule::SendToMobile(
+    const vr_mobile_api::ServiceMessage& message) const {
   LOG4CXX_AUTO_TRACE(logger_);
-  // TODO(KKolodiy): should be implemented
-  return false;
+
+  uint32_t size = message.ByteSize();
+
+  uint8_t* data = new uint8_t[size];
```

Comment:
> A typical declaration for an array in C++ is:
```javascript
type name [elements];
```
NOTE: The elements field within square brackets [], representing the number of elements in the array, must be a constant expression, since arrays are blocks of static memory whose size must be determined at compile time, before the program runs.

[Tutorial - Arrays](http://www.cplusplus.com/doc/tutorial/arrays/)

But we use this way everywhere. :)